src/cljs/rems/language.cljs

```diff
@@ -2,11 +2,22 @@
   (:require [clojure.string :as str]
             [re-frame.core :as rf :refer [reg-event-fx reg-sub]]))
 
+(def ^:private language-cookie-name "rems-user-preferred-language")
+
+(defn get-language-cookie []
+  (when-let [value (.. js/document -cookie)]
+    (keyword (second (str/split value #"=")))))
```

Comment:
there's no cookie parser available?

Reply:
Also, doesn't this assume that there is only one cookie?

Reply:
Quick search for clojurescript cookie returns at least two libraries which both delegate to [goog.net.Cookies](https://google.github.io/closure-library/api/goog.net.Cookies.html).
https://github.com/reagent-project/reagent-utils/blob/master/src/reagent/cookies.cljs
https://github.com/Quantisan/cljs-cookies/blob/master/src/cljs_cookies/core.cljs

Reply:
I prefer not bringing in a new dependency for this simple usage.

Reply:
What about the one cookie assumption? We also have the session cookie.

Reply:
If someone requires complicated cookie usage later, then they can do the refactoring and use an external dependency (or goog library directly)

Reply:
session cookie is http only, so JS should only see this one.

Reply:
`document.cookie` returns a `;`-separated list of all cookies, which this code does not parse. If somebody adds e.g. Google Analytics to their installation, then this code will break.